### PR TITLE
add form.select for grammaars

### PR DESCRIPTION
**Pull request by cheemurakami**
add select box when creating a new grammar.

Screen Shot 2021-01-01 at 6 02 49 PM

### CODE REVIEW

File: app/views/grammars/_form.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   <div class="field">
     <%= form.label :dialect_id %>
-    <%= form.text_field :dialect_id %>
+    <%= form.select :dialect_id, Dialect.all.pluck(:name_en, :id) %>
```
Comment (gdomingu) [inline app/views/grammars/_form.html.erb:16]: Maybe we should sort this alphabetically? What do you think?

Comment (gdomingu) [reply app/views/grammars/_form.html.erb:16]: Checkout the rails `order` method. https://apidock.com/rails/ActiveRecord/QueryMethods/order

File: app/views/grammars/_form.html.erb [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
   <div class="field">
     <%= form.label :dialect_id %>
-    <%= form.text_field :dialect_id %>
+    <%= form.select :dialect_id, Dialect.order(:name_en).pluck(:name_en, :id) %>
```
Comment (gdomingu) [inline app/views/grammars/_form.html.erb:16]: One more thing. Can you add `include_blank: true` to the end of the `select`. 

Should look like this
`form.select :dialect_id, Dialect.order(:name_en).pluck(:name_en, :id), include_blank: true`

File: app/views/grammars/_form.html.erb (hunk 1 above)
Comment (cheemurakami) [reply app/views/grammars/_form.html.erb]: Done.

File: app/views/grammars/_form.html.erb (hunk 2 above)
Comment (cheemurakami) [reply app/views/grammars/_form.html.erb]: Done.

**Review by gdomingu — APPROVED** [review]
Great job! :shipit:

**Pull request merged**